File: 05. Percabangan/E. If Then Case/e_if_then_case.c
#include <stdio.h>
int main()
{
    int N;
    scanf("%d", &N);
    if (N > 9999){
        printf("puluhribuan\n");
    }
    else if (N > 999){
        printf("ribuan\n");
    }
    else if (N > 99){
        printf("ratusan\n");
    }
    else if (N > 9){
        printf("puluhan\n");
    }
    else{
        printf("satuan\n");
    }
    return 0;
}

File: 05. Percabangan/E. If Then Case/e_if_then_case.go
package main

import (
	"fmt"
)

func main() {
	var N int
	fmt.Scanf("%d", &N)
	if N > 9999 {
		fmt.Printf("puluhribuan\n")
	} else if N > 999 {
		fmt.Printf("ribuan\n")
	} else if N > 99 {
		fmt.Printf("ratusan\n")
	} else if N > 9 {
		fmt.Printf("puluhan\n")
	} else {
		fmt.Printf("satuan\n")
	}
}
